pkg/controller/workspace/utils/utilities.go

```diff
@@ -10,12 +10,13 @@
 //   Red Hat, Inc. - initial API and implementation
 //
 
-package workspace
+package utils
```

Comment:
`IsOpenShift` & `IsWebhookConfigurationEnabled` are not really workspace utils...
What do you think if we move somewhere in general place, like
`pkg/k8s/client/utils` and name package as k8sutils?

Reply:
BTW. I've reverted this change since it's not really related to webhook, and we can improve it a separate PR and in some upcoming PR.

Reply:
Sounds good, :+1: for moving generic stuff into another package.

Reply:
`GetContainerNameFromImage` is moved to existing `pkg/controller/modelutils/k8s`

Other like `isOpenShift`, `IsWebhookConfigurationEnabled` are moved into cluster package in `internal/cluster/info.go` file. I'm open for proposal related to better naming.

Reply:
P.S. I also discovered that unused imports which are imported with `.` are not detected as such.
So, we need to try to get rid of them, but I've done it yet to avoid huge merge conflicts with your refactoring.

Reply:
Yeah, I try to remove `.` imports when I can but it results in significant changes. We should try to separate that out into one large PR.